io.openems.edge.timeofusetariff.awattar/src/io/openems/edge/timeofusetariff/awattar/TimeOfUseTariffAwattarImpl.java

```diff
@@ -115,12 +115,13 @@ protected void deactivate() {
 		this.channel(TimeOfUseTariffAwattar.ChannelId.HTTP_STATUS_CODE).setNextValue(httpStatusCode);
 
 		/*
-		 * Schedule next price update for 2 pm
+		 * Schedule next price update every hour
 		 */
 		var now = ZonedDateTime.now();
-		var nextRun = now.withHour(14).truncatedTo(ChronoUnit.HOURS);
+		var nextRun = now.plusHours(1).truncatedTo(ChronoUnit.HOURS);
+
 		if (now.isAfter(nextRun)) {
-			nextRun = nextRun.plusDays(1);
+			nextRun = nextRun.plusHours(1);
```

Comment:
Why is that required? The prices won't change anymore after 2pm, right? We might even meet API (fair use) restrictions if we query to often.

Reply:
This is to ensure a complete schedule for the entire day. Previously, prices were queried at 2 pm, providing 24 values for the next 24 hours. that means at 1:00 pm the next day, we have only one-hour price and the schedule is only done till that hour. 

One alternative way is to query once at 2:00 pm and again at midnight (12:00 am) as well to get prices for the whole day, so that schedule is done for the whole day.